web/src/containers/activity/ContractorResource/ContractorResourceForm.test.js

```diff
@@ -33,200 +33,184 @@ describe('the ContractorResourceForm component', () => {
-      .simulate('change', { target: { value: 773 } });
-
-    expect(props.setCostForYear).toHaveBeenCalledWith(43, 1, '1066', 773);
-  });
-
-  test('handles changing the contractor yearly cost', () => {
-    props.item.hourly.useHourly = true;
-    const parentComponent = shallow(<ContractorForm {...props} />);
-    const component = shallow(
-      parentComponent
-        .findWhere(
-          c => c.name() === 'ChoiceComponent' && c.prop('value') === 'yes'
-        )
-        .prop('checkedChildren')
-    );
-
-    component
-      .findWhere(c => c.prop('name') === 'contractor-num-hours-ffy-1066')
-      .simulate('change', { target: { value: 3752 } });
-
-    expect(props.setNumberOfHoursForYear).toHaveBeenCalledWith(
-      43,
-      1,
-      '1066',
-      3752
-    );
-  });
-
-  test('handles changing the contractor yearly cost', () => {
-    props.item.hourly.useHourly = true;
-    const parentComponent = shallow(<ContractorForm {...props} />);
-    const component = shallow(
-      parentComponent
-        .findWhere(
-          c => c.name() === 'ChoiceComponent' && c.prop('value') === 'yes'
-        )
-        .prop('checkedChildren')
-    );
-
-    component
-      .findWhere(c => c.prop('name') === 'contractor-hourly-rate-ffy-1067')
-      .simulate('change', { target: { value: 9364 } });
-
-    expect(props.setHourlyRateForYear).toHaveBeenCalledWith(
-      43,
-      1,
-      '1067',
-      9364
-    );
-  });
-
-  test('handles changing from non-hourly to hourly and back, replacing cost totals', () => {
-    // start with non-hourly and fill in the total cost
-    props.item.hourly.useHourly = false;
-    const component = shallow(<ContractorForm {...props} />);
-
-    component
-      .findWhere(
-        c =>
-          c.name() === 'DollarField' &&
-          c.prop('name') === 'contractor-cost-ffy-1066'
-      )
-      .simulate('change', { target: { value: '53792' } });
-
-    // now we have the non-hourly cost
-    expect(props.setCostForYear).toHaveBeenCalledWith(43, 1, '1066', '53792');
-
-    // switch to hourly and we have the hourly cost
-    component
-      .findWhere(
-        c =>
-          c.name() === 'ChoiceComponent' &&
-          c.prop('name') === 'apd-activity-contractor-hourly-key 1-yes'
-      )
-      .simulate('change');
-    expect(props.setIsHourly).toHaveBeenCalledWith(43, 1, true);
-    expect(props.setCostForYear).toHaveBeenCalledWith(43, 1, '1066', 1000);
-
-    // switch back to non-hourly and we should have the original cost total
-    component
-      .findWhere(
-        c =>
-          c.name() === 'ChoiceComponent' &&
-          c.prop('name') === 'apd-activity-contractor-hourly-key 1-no'
-      )
-      .simulate('change');
-    expect(props.setIsHourly).toHaveBeenCalledWith(43, 1, false);
-    expect(props.setCostForYear).toHaveBeenCalledWith(43, 1, '1066', '53792');
-  });
+//   test('renders if use hourly data is selected', () => {
```

Comment:
are we going to be using these tests later or can we delete them?

Reply:
good catch, I thought I did delete them